### PR TITLE
New arm parameters   azure feature

### PR DESCRIPTION
**Pull request by RDMacLachlan**

### CODE REVIEW

File: Tools/PowershellModule/src/Library/Add-WinGetManifest.ps1 [hunk 1]
```diff
@@ -93,6 +93,7 @@ Function Add-WinGetManifest
         Write-Verbose -Message "Retrieving a copy of the app Manifest file for submission to WinGet source."
         $ApplicationManifest = Get-WinGetManifest -Path $Path
         if(!$ApplicationManifest) {
+            Write-Verbose "$ApplicationManifest`n`n`n"
```
Comment (zachcarp) [inline Tools/PowershellModule/src/Library/Add-WinGetManifest.ps1:96]: Did you mean to have `n`n`n in this?

File: .gitignore [hunk 2]
```diff
@@ -31,6 +31,12 @@ bld/
 [Ll]og/
 [Ll]ogs/
 
+# These are the PowerShell helper libraries used to run. Added to artifact as automated task
+Tools/PowershellModule/src/Library/ARMTemplate
+Tools/PowershellModule/src/Library/RestAPI
+Tools/PowershellModule/src/Library/WinGet.RestSource.PowerShellSupport
+Tools\PowershellModule\src\Library\WinGet.RestSource.PowershellSupport
```
Comment (zachcarp) [inline .gitignore:38]: Remove last ignore - it isn't formatted correctly

File: Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1 [hunk 3]
```diff
@@ -49,12 +49,39 @@ Function New-ARMParameterObject
...
         $FrontDoorName      = $Name
+        $appConfigName      = $Name
+        $aspGenevaName      = $Name
 
         ## The names of the Azure Cosmos Database and Container - Do not change (Must match with the values in the compiled
         ## Windows Package Manager Functions [WinGet.RestSource.Functions.zip])
         $CDBDatabaseName    = "WinGet"
         $CDBContainerName   = "Manifests"
 
+        ## The values required for Function ARM Template
```
Comment (zachcarp) [inline Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1:60]: The ASP is for internal use - there should be a normal ASP that can be used externally

File: Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1 [hunk 4]
```diff
@@ -440,117 +486,51 @@ Function New-ARMParameterObject
...
-            #                     sendRecvTimeoutSeconds      = 30
-            #                 }
-            #             }
-            #             enabledState = @{
-            #                 value = "Enabled"
-            #             }
-            #         }
-            #     }
-            # }
+            #@{  ObjectType = "ASP_Geneva"
```
Comment (zachcarp) [inline Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1:505]: Do you need all this commented out? If not - please remove

File: Tools/PowershellModule/src/Microsoft.WinGet.Source.psd1 [hunk 5]
```diff
@@ -3,7 +3,7 @@
 #
 # Module manifest for module 'Microsoft.WinGet.Source'
 #
-# Generated by: denelon
+# Generated by: romaclac
 #
 # Generated on: 10/15/2021
```
Comment (zachcarp) [inline Tools/PowershellModule/src/Microsoft.WinGet.Source.psd1:8]: Update date

**Review by zachcarp — CHANGES_REQUESTED** [review]

File: Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1 (hunk 3 above)
Comment (zachcarp) [reply Tools/PowershellModule/src/Library/New-ARMParameterObject.ps1:60]: Disregard this

File: Tools/PowershellModule/src/Library/Add-WinGetManifest.ps1 (hunk 1 above)
Comment (RDMacLachlan) [reply Tools/PowershellModule/src/Library/Add-WinGetManifest.ps1:96]: This is intentional. New Line character.

**Review by zachcarp — APPROVED** [review]

**Pull request merged**